modules/utils/src/string/string_utils.cpp

```diff
@@ -6,18 +6,14 @@
 
 #include <algorithm>
 #include <cctype>
-#include <charconv>
-#include <cstdint>
 #include <cstdio>
 #include <iterator>
-#include <optional>
 #include <string>
 #include <string_view>
-#include <system_error>
 
 namespace heph::utils::string {
 
-auto toUpperCase(const std::string_view& any_case) -> std::string {
```

Comment:
also for those we could use https://github.com/abseil/abseil-cpp/blob/master/absl/strings/ascii.h?